sim/hunter/item_sets_pve.go

```diff
@@ -200,17 +200,21 @@ var ItemSetDragonstalkerProwess = core.NewItemSet(core.ItemSet{
 			hunter := agent.(HunterAgent).GetHunter()
 
 			procAura := hunter.RegisterAura(core.Aura{
-				ActionID: core.ActionID{SpellID: 467329},
-				Label:    "S03 - Item - T2 - Hunter - Melee 2P Bonus",
+				ActionID: core.ActionID{SpellID: 467331},
+				Label:    "Clever Strikes",
 				Duration: time.Second * 5,
 				OnGain: func(aura *core.Aura, sim *core.Simulation) {
 					for _, spell := range hunter.MeleeSpells {
-						spell.DamageMultiplier *= 1.20
+						if spell.SpellCode != (SpellCode_HunterRaptorStrikeHit | SpellCode_HunterRaptorStrike) {
+							spell.DamageMultiplier *= 1.20
+						}
 					}
 				},
 				OnExpire: func(aura *core.Aura, sim *core.Simulation) {
 					for _, spell := range hunter.MeleeSpells {
-						spell.DamageMultiplier /= 1.20
+						if spell.SpellCode != (SpellCode_HunterRaptorStrikeHit | SpellCode_HunterRaptorStrike) {
```

Comment:
The base raptor strike spell doesn't actually deal any damage so we should be able to just use `SpellCode_HunterRaptorStrikeHit` here